src/app/work-item/work-item-list/work-item-list.component.html

```diff
@@ -1,70 +1,18 @@
-<div *ngIf="showDialog">	
-	<pf-dialog
-		(pfDialogClick)="onButtonClick($event)"
-		[dialog]='dialog'	
-	></pf-dialog>
-</div>
 <app-header></app-header>
 <div class="container-fluid screen-min-ht containerPaddT">
 	<div id="workItemList_quickAdd" class="width100 fl">
 		<div class="width100 fl supress-hover" (click)="addWorkItem()">
 			<work-item-quick-add (close)="close($event)"></work-item-quick-add>
 		</div>
-	</div>
-	<div class="list-group list-view-pf width100 fl">
-		<div *ngFor="let workItem of workItems; let counter = index" id="{{'workItemList_OuterWrap_'+counter}}" class="work-item-list-entry">
-			<div class="list-group-item" (click)="onSelect(workItem)"
-					 [class.selected]="workItem === selectedWorkItem">
-				<div class="list-view-pf-actions">
```

Comment:
I'm going to guess that this section got missed in a merge. Please add it back in.